### DISCUSSION TITLE
convert calendar type to solar or jalali system

### DISCUSSION THREAD

**Discussion by ardev0** [liamcain/obsidian-calendar-plugin#238] [Q&A]
Hello
Is there a way to change this calendar from Gregorian to solar(jalali)?
Which classes are related to making the calendar look and structure and the order of the day and month.
Because in addition to the months, the beginning and end of each month are different in these two calendar models.
please guide me.
Is it possible to use the region area in Windows for the calendar without the need for much coding?

thanks for support and helping people for better life;)

**Comment by amirmeimari** [comment]
jalali should be there. lots of people are using it.

**Comment by ShahriarKh** [comment]
Obsidian uses moment.js and so does this plugin. moment.js doesn't support Jalali (Shamsi) calendar, so we need to somehow implement a library like https://github.com/jalaali/moment-jalaali or just use `Intl` (described well here: https://github.com/jalaali/moment-jalaali#note-feb-2022)

**Reply by ShahriarKh** [reply]
similar issue: #295
